### ISSUE TITLE
_handleClick being called twice

### ISSUE THREAD

**Issue by Hellysonrp** [Maxl94/ngx-multi-sort-table#27]
`MatMultiSortHeaderComponent` `_handleClick` is being called twice: one for the `host` of the [MatSortHeader](https://github.com/angular/components/blob/bb2bb9c3065ec1a7a845de072305893a7378383a/src/material/sort/sort-header.ts#L81) and another for the `_handleClick` [HostListener decorator](https://github.com/Maxl94/ngx-multi-sort-table/blob/96da2f03044e0b0d8275ac22f29e8035b8ba75ad/projects/mat-multi-sort/src/lib/mat-multi-sort-header/mat-multi-sort-header.component.ts#L58).

This behavior is related to [this change](https://github.com/angular/angular/issues/36563) in Angular.

![Screenshot_2](https://user-images.githubusercontent.com/5438762/99915304-a99f8680-2ce1-11eb-9e02-6b24e7c5170d.png)

![Screenshot_3](https://user-images.githubusercontent.com/5438762/99915305-aad0b380-2ce1-11eb-85f3-7e535d29e8de.png)

AFAIK you can safely remove the `HostListener` from the `MatMultiSortHeaderComponent`, but I'm not certain.

Angular CLI version: 10.2.0
Angular version: 10.1.7

**Comment by Maxl94** [comment]
Thanks for your update. I currently working at a bug, where the sort directions is set from None -> DESC which makes no senes. Maybe this because of click handled dupplicated. I am not shure right now.

**Comment by Hellysonrp** [comment]
@Maxl94 

See my PR #28.
I might have fixed that.
It need some test though.

**Comment by Maxl94** [comment]
Jup, seams so. I added your changes to my branch it's working as expecated.

**Comment by Maxl94** [comment]
@Hellysonrp when do you think you have finished the tests? I like to create a new release with all these fixes.

**Comment by Hellysonrp** [comment]
For my use case, everything seems to be working correctly now.
I did not find any other issue since yesterday. However, I'm not using the `TableData`, neither the `MatMultiSortTableDataSource`.
I'm using my own implementation of a `DataSource` because I already had a `MatSort` implemented. I just adapted my code to use `MatMultiSort`.

I've tested the `TableData`, `MatMultiSortTableDataSource`, and `MatMultiSortTableSettingsComponent` on the demo project and didn't find any issues.
I'm confident that everything is working correctly.
